### PR TITLE
TM-8746 Container Settings API documentation

### PR DESCRIPTION
**Pull request by bua89**
- added the prototype of API documentation

### CODE REVIEW

**Review by stiopanoff — APPROVED** [review]

**Comment by JacekClearcode** [comment]
Never chose `master` as the target for PR. Should it be 16.15?

File: _templates/layout.html [hunk 1]
```diff
@@ -52,6 +52,7 @@
         'audience_manager',
         'consent_manager',
         'tag_manager',
+        'container_settings',
```
Comment (JacekClearcode) [inline _templates/layout.html:55]: Please refrain from making changes here, unless you are adding stuff to the main menu in the top. This change is now messing it up.

**Review by JacekClearcode — CHANGES_REQUESTED** [review]

**Comment by JacekClearcode** [comment]
[Built version](https://developers.piwik.pro/en/feature-tm-8746-container-settings-api-documentation/platform/authorized_api/container_settings/container_settings_api.html#/paths/~1api~1container-settings~1v1~1app~1{app_id}~1settings/get)

File: _templates/layout.html (hunk 1 above)
Comment (bua89) [reply _templates/layout.html]: Fixed

**Comment by bua89** [comment]
@JacekClearcode could you verify fixes?

**Comment by JacekClearcode** [comment]
https://github.com/PiwikPRO/PPMS-PublicDocs/pull/851#issuecomment-1077166945 - I asked about this as well.

**Comment by bua89** [comment]
@JacekClearcode could you provide a branch for version 16.16?

**Comment by JacekClearcode** [comment]
Yup. Done

**Review by JacekClearcode — APPROVED** [review]

**Comment by bua89** [comment]
Could you also generate rendered documentation for 16.16 branch? It would help to develop changes related to the automatic scroll tracking.

**Pull request merged**